tests/bluetooth/tester/src/gatt.c

```diff
@@ -1534,10 +1535,10 @@ static void read_multiple(uint8_t *data, uint16_t len)
 	read_params.func = read_cb;
 	read_params.handle_count = i;
 	read_params.multiple.handles = handles; /* not used in read func */
-	read_params.multiple.variable = false;
+	read_params.multiple.variable = variable_length;
```

Comment:
And this would become `read_params.multie.variable = (opcode == GATT_READ_MULTIPLE_VAR);`